test/api/test_rest.py

```diff
@@ -468,6 +471,10 @@ def test_testruns(self):
         data = self.hit('/api/testruns/%d/' % self.testrun.id)
         self.assertEqual(self.testrun.id, data['id'])
 
+    def test_testruns_filter(self):
+        data = self.hit('/api/testruns/?completed=%s&build__id=%s' % (self.testrun.completed, self.build4.id))
```

Comment:
You declared `self.testrun4` but used `self.testrun` instead. It worked because TestRun objects have `completed=True` by default (https://github.com/Linaro/squad/blob/master/squad/core/models.py#L609)

Reply:
Thanks @chaws for catching this. Fixed and repushed.